internal/run/target.go

```diff
@@ -112,6 +112,15 @@ func (w *Workflow) runTarget(ctx context.Context, target string) (*SourceResult,
 		return sourceRes, nil, err
 	}
 
+	targetResult := TargetResult{
+		OutputPath:  outDir,
+		GenYamlPath: genConfig.ConfigPath,
+	}
+	defer func() {
+		w.SourceResults[sourceRes.Source] = sourceRes
```

Comment:
I feel like this should be set during runSource

Reply:
ah sorry it is thanks